specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2022-08-01/definitions.json

```diff
@@ -233,6 +233,60 @@
       },
       "description": "Paged API list representation."
     },
+    "DocumentationCollection": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DocumentationContract"
+          },
+          "x-ms-identifiers": [
+            "name"
+          ],
+          "description": "Page values.",
+          "readOnly": true
+        },
+        "count": {
```

Comment:
Do not return count  [https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#collections](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#collections)

Reply:
![image](https://user-images.githubusercontent.com/11067581/213276353-8204061f-acbe-4e01-9b83-232af207f4b7.png)

Reply:
removed